### PR TITLE
Add note about content guidelines in writing guidelines page

### LINKED ISSUE #4066: Cross-link between content guidelines and other docs guidelines
In community/contributing/documentation_guidelines.rst, we should link to the content guidelines. 

Depends on #4064.

### PR DESCRIPTION
**Pull request by skyace65**
Adds a note to the beginning of the writing guidelines page about the content guidelines. Closes #4066

### CODE REVIEW

**Comment by pycbouh** [comment]
I'm not entirely sure what would be the best place for this note. Maybe some clean up is required in the article itself to make that cross-link more fitting. I guess for the time being we can add it like this.

**Review by pycbouh — APPROVED** [review]

File: community/contributing/docs_writing_guidelines.rst [hunk 1]
```diff
@@ -32,6 +32,11 @@ There are 3 rules to describe classes:
     the smallest and clearest sentences possible. These guidelines will help
     you work towards that goal.
 
+.. note::
+
+    For information on what type of documentation you can create see the
+    `content guidelines <doc_content_guidelines>`__.
```
Comment (Calinou) [inline community/contributing/docs_writing_guidelines.rst:38]:
```suggestion
.. seealso::

    See the `content guidelines <doc_content_guidelines>`__ for information
    on the types of documentation you can write.
```

**Review by Calinou — CHANGES_REQUESTED** [review]

**Comment by skyace65** [comment]
Huh, I didn't even know we had a "see also" thing. Fixed.

**Comment by NathanLovato** [comment]
Thanks!

In general, I think it's good to cross-ref in different places so the users has multiple paths to stumble pages they're interested in, as it depends a lot on how they land on the docs and their browsing habits.

**Pull request merged**